base_layer/wallet/src/output_manager_service/mod.rs

```diff
@@ -106,7 +106,7 @@ where T: OutputManagerBackend + 'static
         );
 
         let (sender, receiver) = reply_channel::unbounded();
-        let (publisher, _) = broadcast::channel(200);
+        let (publisher, _) = broadcast::channel(3500);
```

Comment:
shouldn't this use the new config?

Reply:
It was actually only the transaction event channel size that was made configurable due to stats in the description; do you suggest having more of these configurable?

Reply:
Ja, if we are doing it then I think we should do it for both.

Reply:
Cool, making all those broadcast channels sizes configurable in the config file.